### ISSUE TITLE
cmake CMakeList.txt on Linux failed

### ISSUE THREAD

**Issue by chraibi** [JuPedSim/jpsvis#53]
In Gitlab by @ghost on Nov 25, 2017, 11:44 [[origin](https://gitlab.version.fz-juelich.de/jupedsim/jpsvis/issues/53)]

To fix this bug, try this:

On Line 11:
```
cmake_policy(SET CMP0025 NEW) #fixes error No known features for CXX compiler (on mac)
```

Change it to this one:
```
IF(APPLE)
   cmake_policy(SET CMP0025 NEW) #fixes error No known features for CXX compiler (on mac)
ENDIF(APPLE)
```
After this change you can compile on Linux (Tested on Ubuntu 14.04)

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Feb 8, 2018, 15:12

changed the description

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Feb 8, 2018, 15:12

changed the description

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Feb 8, 2018, 15:16

closed via commit 3ecb53167c2cd1bd2b536287efdbdb3501cf37de
